<launch>

    <!-- Node for sending images from bag -->
	<arg name="video_folder_prefix" default="" />
	<node name="convert_rgb_to_depth" pkg="franka_irom" type="convert_rgb_to_depth.py" output="screen" args="$(arg video_folder_prefix)"/>

	<!-- Depth to point cloud conversion -->
    <!-- http://wiki.ros.org/depth_image_proc#depth_image_proc.2BAC8-point_cloud_xyz -->
    <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
    <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/register nodelet_manager">
        <remap from="rgb/camera_info" to="depth/camera_info"/>
        <remap from="depth/camera_info" to="rgb/camera_info"/>
        <!-- <remap from="rgb_for_conversion" to="depth/image_rect"/> -->
        <remap from="depth/image_rect" to="rgb_for_conversion"/>
    </node>

    <!-- Node for publishing static transform between depth and base -->
    <node pkg="tf" type="static_transform_publisher" name="depth_to_rgb" args=" -0.032 -0.002 0.004 -0.051 0.000 0.004 0.999 depth_optical_frame rgb_optical_frame 100"/>

</launch>

